src/theme/sass-partials/_forms.scss

```diff
@@ -0,0 +1,43 @@
+$checkbox-width: 30px;
+input[type='checkbox'] {
+  transform: scale(2);
+  margin: 25px 20px;
+  & + label {
+    font-size: 20px;
+    position: relative;
+    cursor: pointer;
+    margin: 20px 2px;
+  }
+}
+
+.signature-pad {
+  button#retry {
+    float: left;
+  }
+  button#continue {
+    float: right;
+  }
+  background-color: #fafafa;
```

Comment:
We can create new colours in _variables.scss which will help us to re-use the same ones and keep track of our overall colour pallette